gcc/build/build.sh

```diff
@@ -113,15 +113,11 @@ else
     BINUTILS_DIR=binutils-${BINUTILS_VERSION}
     rm -rf ${BINUTILS_DIR}
     tar jxf binutils-${BINUTILS_VERSION}.tar.bz2
-    BINUTILS_FILES=$(cd ${BINUTILS_DIR}; ls -1)
-    pushd gcc-${VERSION}
-    for file in ${BINUTILS_FILES}
```

Comment:
I'm so glad to be rid of this heinous hack.

Reply:
I think there's a script in the binutils source that does something similar, `symlink-tree`